rustica/src/logging/mod.rs

```diff
@@ -41,6 +41,8 @@ pub struct CertificateIssued {
     pub fingerprint: String,
     /// The fingerprint of the signing certificate
     pub signed_by: String,
+    /// The serial number of the signing certificate
```

Comment:
The serial number of the signed certificate would be more accurate. Since we're not signing from an existing certification anyway, "serial number of the sign*ing* certificate could be confusing.